### PR TITLE
Use docker_image instead of docker pull

### PR DESCRIPTION
**Pull request by varac**
With a command 'docker pull' the resource always change.

### CODE REVIEW

**Comment by mhutter** [comment]
Hmm, this has two drawbacks:
* `docker-py` is induced as an additional dependency (I found that it is usually missing on fresh systems) so we'd have to ensure it is installed before running `docker_image`

* `docker_image` will not update images that are already present on the system but might have changed on docker hub (eg. when using the `latest` tag). This can be enforced by setting `force` to true but this will in turn always mark the resource as changed IIRC... but then again, you should use versioned tags anway...

WDYT?

**Comment by varac** [comment]
> docker-py is induced as an additional dependency (I found that it is usually missing on fresh systems) 
> so we'd have to ensure it is installed before running docker_image

Agreed, I found that having this drawback is worth having the advantage that the resource won't change on every run. I just wonder if we should document this dependency and offload it to the user how to install it, or create a package resource that would install it (but then, it might have a different name on different OSes...).

> 
> docker_image will not update images that are already present on the system but might have changed on docker hub (eg. when using the latest tag).
> This can be enforced by setting force to true but this will in turn always mark the resource as changed IIRC... but then again, you should use versioned tags anway...

No, using `force: True` only will only change the resource when an actual update happens. I just tried it to make sure, and pushed an update with `force: True`.

File: tasks/install.yml [hunk 1]
```diff
@@ -9,9 +9,10 @@
   when: container_env is defined
   notify: restart container {{ container_name }}
 
-# use `command` instead of `docker_image` so we don't have to install docker-py
 - name: Pull image {{ container_image }}
-  command: docker pull {{ container_image }}
+  docker_image:
+    name: '{{ container_image }}'
+    force: True
```
Comment (mhutter) [inline tasks/install.yml]: Use `true`

**Review by mhutter — CHANGES_REQUESTED** [review]
Can you also document the `docker-py` in the README please

**Review by mhutter — APPROVED** [review]

**Pull request merged**